docs/application/dotnet/guides/nui/resources.md

```diff
@@ -28,7 +57,7 @@ The following formats are supported:
 | WAP Bitmap              | `.wbmp`    |
```

Comment:
I noticed that lines 46-56 are merged and cannot be commented out. Not sure what this implies here?